gms/impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -121,32 +128,46 @@
 
   @Action(name = ACTION_INGEST)
   @Nonnull
-  public Task<Void> ingest(@ActionParam(PARAM_ENTITY) @Nonnull Entity entity) throws URISyntaxException {
+  public Task<Void> ingest(
+      @ActionParam(PARAM_ENTITY) @Nonnull Entity entity,
+      @ActionParam(SYSTEM_METADATA) @Optional @Nullable SystemMetadata systemMetadata
+  ) throws URISyntaxException {
+
+    if (systemMetadata == null) {
+      SystemMetadata generatedSystemMetadata = new SystemMetadata();
+      generatedSystemMetadata.setLastObserved(System.currentTimeMillis());
```

Comment:
Oh got it - so client can optionally overwrite this. It still may make sense to keep last observed consistent by overriding this on the GMS side.. thoughts?

Reply:
good call- i already have logic to handle system metadata not being set in gms. Updating so the client sends a null system metadata if it doesn't receive one.